packages/manager/apps/dedicated-servers/src/components/exportCsv/constants.ts

```diff
@@ -0,0 +1,11 @@
+export const TITLE = {
+  MAIN: 'server_export_csv_title',
+  COMPLETED: 'server_export_csv_export_completed_title',
+};
+
+export enum ExportCsvStateEnum {
+  initial = 'initial',
+  exportConsent = 'exportConsent',
+  onExportCsv = 'onExportCsv',
+  exportCompleted = 'exportCompleted',
```

Comment:
I think that by convention, enum values should be in serpent case:
```suggestion
  INITIAL = 'initial',
  EXPORT_CONSENT = 'exportConsent',
  ON_EXPORT_CSV = 'onExportCsv',
  EXPORT_COMPLETED = 'exportCompleted',
```

Reply:
done thx